guides/common/modules/proc_assigning-resource-quotas-to-a-user.adoc

```diff
@@ -11,3 +12,29 @@ You can assign resource quotas to a user to limit the resource consumption of th
 +
 This means that {Project} will not prevent users from consuming more resources than their assigned quota.
 . Click *Submit* to assign resource quotas to a user.
+
+[id="cli-assigning-resource-quotas-to-a-user"]
+.CLI procedure
+. Optional: List all {Project} users:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer user list --fields id,login
+----
+. Optional: List all resource quotas:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer resource-quota list --fields id,name
+----
+. Assign resource quotas to a user:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer user update \
+--id _My_User_ID_ \
+--resource-quota-ids _My_Resource_Quota_IDs_ \
+--resource-quota-is-optional _[true|false]_
```

Comment:
Would you accept this? Is there a more idiomatic way of saying "either this or that, but not both or none"?

Reply:
How about choosing one and making clear that it's an example? Like this:

"Assign resource quotas to a user. For example, to [...] while making resource quotas optional:
[...]
$ hammer user update [...] --reqource-quota-is-optional true"

And then you already refer users to --help, which gives them a clear signal where to go for all the other details.

Reply:
I have moved it below the code block and marked it as conditional.